libgadget/gravshort-tree.c

```diff
@@ -98,6 +140,10 @@ void grav_short_tree(ForceTree * tree)
 
     walltime_add("/Tree/Misc", timeall - (timetree + timewait + timecomm));
 
+    /* TreeUseBH > 1 means use the BH criterion on the initial timestep only,
+     * avoiding the fully open O(N^2) case.*/
+    if(TreeParams.TreeUseBH > 1)
```

Comment:
It's a bit confusing to mutate the Param during runtime.

Reply:
True. But I could not think of a cleaner way to do it.